content/open-source/wolfi/hello-wolfi.md

```diff
@@ -0,0 +1,120 @@
+---
+title: "Hello Wolfi Workshop Kit"
+type: "article"
+description: "Community workshop kit about Wolfi for beginners"
+date: 2022-12-19T08:49:31+00:00
+lastmod: 2022-12-19T08:49:31+00:00
+draft: false
+images: []
+menu:
+  docs:
+    parent: "wolfi"
+weight: 500
+toc: true
+---
+
+
+This workshop kit provides resources for individuals who would like to present a talk or workshop about Wolfi and the tools that comprise its ecosystem.
+
+The following materials are included:
+
+- [Presentation Slides](https://docs.google.com/presentation/d/e/2PACX-1vRBWi-C8w1h8UC2Kd6eYd1kyT4Qsdky9Z8X6frLoPJNeKMGZ_v7Ev4Lkfqq1I_pQIWVpHrRXKPBvcuH/pub?start=false&loop=false&delayms=3000);
+- A [YouTube Video](https://www.youtube.com/watch?v=2pqhLXA6NaI) with the original talk presented at the [Wolfi 101 Webinar](https://www.crowdcast.io/c/wolfi-101);
+- A [GitHub repository](https://github.com/chainguard-dev/hello-wolfi-demo) containing all files necessary to build and execute a demo showcasing Wolfi, melange, and apko;
+- This guide, which includes instructions on how to execute the demo.
+
+{{< youtube 2pqhLXA6NaI >}}
+
+## Introduction
+
+Software supply chain threats have been growing exponentially in the last few years, according to [industry leaders and security researchers (PDF)](https://www.usenix.org/system/files/login/articles/login_winter20_17_geer.pdf).
+With the popularization of automated workflows and cloud native deployments, it is more important than ever to provide users with the ability to attest the provenance of all relevant software artifacts that compose the container images being used as build and production runtimes.
+
+In this workshop, you'll learn more about Wolfi, a community Linux _undistro_ designed for the container and cloud-native era. You'll also learn about melange and apko, Chainguard's open source toolkit created to build more secure container images.
+
+## Running the Demo
+
+The demo files are available on the [chainguard.dev/hello-wolfi-demo](https://github.com/chainguard-dev/hello-wolfi-demo) repository. These steps were executed on an Ubuntu 22.04 host Linux machine.
+
+The demo application is a command-line script built with PHP. It connects to the [cat facts](https://catfact.ninja/) API and outputs a quote about cats.
+
+Before getting started, make sure you have Docker installed on your machine. The demo consists of the following steps:
+
+1. Download the `cgr.dev/chainguard/melange` and `cgr.dev/chainguard/apko` images with `docker pull`
+2. Generate melange signing keys
+3. Build the `melange-php.yaml` package
+4. Build the `composer-php.yaml` package
+5. Build the `melange-app.yaml` package
+6. Build the `apko.yaml` container image
+7. Load the image with `docker load`
+8. Run the image with `docker run --rm <image-name>`
+
+### 1. Download melange and apko images
```

Comment:
Would you be against adding a `git clone https://github.com/chainguard-dev/hello-wolfi-demo` and `cd hello-wolfi-demo` step before launching in here?

Reply:
no problem, I will add that!